update.go

```diff
@@ -150,6 +153,9 @@ var updateManifestRetrying = false
 func GetUpdateManifest(channel string) *Manifest {
 	var m Manifest
 	url := "https://cli-assets.heroku.com/branches/" + channel + "/manifest.json"
+	if config.LockVersion != "" {
```

Comment:
@dickeyxxx what would you think of adding a `version string` argument and moving the usage of `config.LockVersion` up into `updateCLI` so that all the config stuff is localized?